tutorials/02_configuration.md

```diff
@@ -62,13 +62,13 @@ mkdir /tmp/conf_tutorial && cd /tmp/conf_tutorial
 Download the file `download.cc` and save it under `/tmp/conf_tutorial`:
 
 ```
-wget https://github.com/ignitionrobotics/ign-fuel-tools/raw/ign-fuel-tools4/example/download.cc
+wget https://github.com/ignitionrobotics/ign-fuel-tools/raw/master/example/download.cc
```

Comment:
This should probably be `ign-fuel-tools5` for consistency, right?

Reply:
There is no branch for `ign-fuel-tools5`.

Reply:
Done in cee8927941175ab920dae47b5f3758eba2d1d911